.github/workflows/galaxy_ci.yml

```diff
@@ -110,7 +110,7 @@ jobs:
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": "Galaxy Installer CI ${{ job.status }}\nhttps://github.com/pulp/pulp_installer/runs/${{ github.run_id }}?check_suite_focus=true",
+                    "text": "Galaxy Installer CI ${{ job.status }}\nhttps://github.com/pulp/pulp_installer/runs/${{ github.run_id }}?check_suite_focus=true"
```

Comment:
I would prefer if we didn't spam the channel with failed PR-CI runs. For a pull request, there is usually someone waiting and therefore seeing the failure. I can for sure see a benefit to notify about failed nightlies. These tend to go unnoticed way too often.

Reply:
Great!